ci/test/cargo-test/mzcompose.py

```diff
@@ -141,7 +141,6 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
                     f"--partition=count:{partition}/{total}",
                     # Most tests don't use 100% of a CPU core, so run two tests per CPU.
                     # TODO(def-): Reenable when #19931 is fixed
-                    f"--test-threads={cpu_count // 2}",
                     # f"--test-threads={cpu_count * 2}",
```

Comment:
Do you want to enable the commented-out line? Or keep the default, which I assume is test-threads=cpu_count  ? Or the default is 1 thread?

Reply:
I want the default, 1 per CPU thread.